styles/style.css

```diff
@@ -0,0 +1,382 @@
+* {
+    font-size: 1.15rem;
+    margin: 0;
+}
+
+header {
+    background-image: url("../images/header-image.jpg");
+    background-position: right;
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+
+}
+
+header h1 {
+    color: rgb(18, 146, 18);
+    font-size: 1.25rem;
+}
+
+header p {
+    color: white;
+}
+
+.header-left {
+    padding-left: 6%;
+    padding-top: 2%;
+    padding-bottom: 10%;
+
+}
+
+.header-right {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    padding-right: 6%;
+    padding-top: 2%;
+    padding-bottom: 10%;
+}
+
+.cart {
+    width: 20%;
+}
+
+.pay {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 5%;
+    justify-content: flex-end;
+    align-content: center;
+}
+
+.pay p {
+    margin-top: 10%;
+}
+.pay img {
+    margin-right: 5%;
+}
+
+nav {
+    background-color: lightgray;
+}
+
+nav a,
+footer a {
+    text-decoration: none;
+    font-weight: bold;
+}
+
+nav li,
+footer li {
+    list-style-type: none;
+    padding: 1%;
+
+}
+
+nav ul {
+    margin-left: 12%;
+
+}
+
+nav ul,
+footer ul {
+    display: flex;
+    flex-direction: row;
+
+}
+
+h2 {
+    font-size: 1.5rem;
+    padding-top: 2%;
+    padding-bottom: 2%;
+    padding-left: 15%;
+}
+
+button {
+    text-align: center;
+    color: white;
+    background-color: rgb(18, 146, 18);
+    font-weight: bold;
+}
+
+.dogs button {
+    margin-left: 10%;
```

Comment:
generally you'll want the same padding each side  horizontally and the same padidng vertically; 

you can shorthand this (using your values...) to
`margin: 7% 10%` which will do 7% top/bottom and 10% left/right.

I'd also suggest  using REM's for padding/margin